### PR TITLE
Tr2 5

### PR DESCRIPTION
**Pull request by Ordash**
-Fixed tests to use only junit 5
-The logic from the controllers moved to service
-implemented method to return the right kingdom name

### CODE REVIEW

**Review by starlinginferno — APPROVED** [review]
Cool

File: src/main/java/rueppellii/backend2/tribes/kingdom/Kingdom.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 import javax.persistence.*;
```
Comment (csadesz) [inline src/main/java/rueppellii/backend2/tribes/kingdom/Kingdom.java:8]: Try to not import all the elements of the library, just the necessary ones

**Review by csadesz — CHANGES_REQUESTED** [review]
Tests are not working on your branch. There are the old junit jupiter ones ( same as in the develop, it isnt working too )

**Comment by giba86** [comment]
Nice. Lots of code very nice implementation of token based authentication. Thanks

File: src/main/java/rueppellii/backend2/tribes/kingdom/Kingdom.java (hunk 1 above)
Comment (Ordash) [reply src/main/java/rueppellii/backend2/tribes/kingdom/Kingdom.java:8]: Jupiter is the latest version thats why I removed everything related to the previous versions and it was working perfectly.

**Review by csadesz — APPROVED** [review]

**Pull request merged**